docker/init.sh

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 
 echo 'Starting PrairieLearn...'
+
+make -s -C /PrairieLearn start-support
+cd /PrairieLearn
```

Comment:
the `-C` option changes the directory where `make` operates. If we're already changing the working directory anyway, it's not necessary.
```suggestion
cd /PrairieLearn
make -s start-support
```

Same for the `make` calls below.

Reply:
OK, I like that. It's cleaner.